docs/source/model/dependencies.rst

```diff
@@ -438,15 +445,15 @@ To do so, use the **pip-requirements-override** option to specify pip dependenci
         import mlflow
 
         mlflow.models.predict(
-            model_uri="runs:/<run_id>/model",
+            model_uri="runs:/<run_id>/<model_path>",
             input_data=<input_data>,
             pip_requirements_override=["opencv-python==4.8.0"],
         )
 
     .. code-tab:: bash
 
         mlflow models predict \
-            -m runs:/<run_id>/model \
+            -m runs:/<run_id>/<model_path> \
             -i <input_path> \
```

Comment:
Could you add a quick note that the file pointed by `input_path` has to be a JSON file that contains the prediction payload, or a CSV file for pandas Dataframe when the user specifies `--content-type csv`?

Reply:
I updated the CLI docs. Does that work?

Reply:
Added a note here as well. Good call